src/lib/api/base.js

```diff
@@ -9,37 +9,70 @@ const apiConfig = {
   xsrfCookieName: 'csrftoken',
   xsrfHeaderName: 'HTTP_X_CSRFTOKEN',
 };
+const http = axios.create(apiConfig);
 
+const HTTP_STATUS_CODES_WITH_ERROR_PAGE = [404, 429, 500];
 const URLS_NOT_TO_REDIRECT_IF_UNAUTHORIZED = ['/me', '/me/loans'];
+const CSRF_ERROR_REASON_NO_COOKIE = 'CSRF cookie';
+const CSRF_ERROR_REASON_BAD_TOKEN = 'CSRF token';
```

Comment:
LGTM! Just one minor question,does it makes sense to add the [error](https://github.com/inveniosoftware/invenio-rest/blob/master/invenio_rest/errors.py#L114) caused by the [expiration of the token](https://github.com/inveniosoftware/invenio-rest/blob/master/invenio_rest/csrf.py#L81) ? 
By adding something like `const CSRF_ERROR_REASON_TOKEN_EXPIRED = 'CSRF error';` we would retry the request instead of showing the `CSRF error` (I guess the users will not understand it) and it would be "transparent" for the user. WDYT?

Reply:
Isn't that happening if you are tampering the token? Then you are probably doing something malicious... do you see any other use case?

Reply:
If you leave it open for 1 hour and you come back the csrf token is expired, or if you only do get requests(for 1 hour) it will never get reloaded, it's not a common situation but I guess it could happen.